ext/configuration.h

```diff
@@ -112,6 +112,7 @@ enum ddtrace_dbm_propagation_mode {
     CONFIG(INT, DD_TRACE_X_DATADOG_TAGS_MAX_LENGTH, "512")                                                     \
     CONFIG(MAP, DD_TRACE_PEER_SERVICE_MAPPING, "")                                                             \
     CONFIG(BOOL, DD_TRACE_PEER_SERVICE_DEFAULTS_ENABLED, "false")                                              \
+    CONFIG(BOOL, DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED, "false")                                   \
```

Comment:
May I note that _remove_ + _enabled_ is quite redundant? `DD_TRACE_INTEGRATION_SERVICE_NAMES_ENABLED=true` would make more sense to me.

Reply:
I agree with you, this is a battle that I fought (and lost) myself 😄. On top of that, there is a negation (remove*) and an affirmation (enable) in the same name, which is also a bad practice in my opinion.
I have to resolve this comment without changes for consistency with the whole project, but I feel what you say.